kotlin-extension-functions/build.gradle

```diff
@@ -1,39 +1,46 @@
-apply plugin: 'org.jetbrains.dokka'
-apply plugin:"kotlin-kapt"
+plugins {
+    id 'io.micronaut.build.internal.module'
```

Comment:
Looks like you have at least 2 projects which use the same combination of plugins. This speaks for adding a convention plugin in `buildSrc`.